### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by Glenson1** [apohl79/audiogridder#395] [General]
Hi,

I want to say thanks to the developer of audiogridder! 
My mac is pretty decent and I can run a fair amount of Acustica plugins. But not enough. So Freezing and printing was necessary. With audiogridder (on the same mac) I can run so much more. 

Fantastic! I will donate asap :-)
Glenn

**Comment by CaBuffalo** [comment]
Yes, this is what AG exactly does.

Thanks for sharing. Cheers.

On Sat, Feb 20, 2021 at 12:23 PM Glenson1 <notifications@github.com> wrote:

> Hi,
>
> I want to say thanks to the developer of audiogridder!
> My mac is pretty decent and I can run a fair amount of Acustica plugins.
> But not enough. So Freezing and printing was necessary. With audiogridder
> (on the same mac) I can run so much more.
>
> Fantastic! I will donate asap :-)
> Glenn
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/395>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FQEHMT4GVNMAAM2PDTAAK45ANCNFSM4X6GZLTQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by alectroniq** [comment]
Mega happy that I found AG by accident - this is what I'm looking for since over 15 yrs. Thanks so much for your excellent work @apohl79 !
Just donated and will donate more in the future as there are more updates. Keep up please 🙏🏼

**Comment by CaBuffalo** [comment]
Thank you very much.

On Sat, Mar 13, 2021 at 7:15 AM alectroniq ***@***.***> wrote:

> Mega happy that I found AG by accident - this is what I'm looking for
> since over 15 yrs. Thanks so much for your excellent work @apohl79
> <https://github.com/apohl79> !
> Just donated and will donate more in the future as there are more updates.
> Keep up please 🙏🏼
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/395#discussioncomment-478147>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HB5CANGTWB53F23Z3TDN6S3ANCNFSM4X6GZLTQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
